utility_code/export_loot_table_items.py

```diff
@@ -125,6 +125,18 @@ def get_refined_creative_tab_flags(item_tag):
 
 out_name = sys.argv[1]
 refined_creative_inventory_folder = sys.argv[2]
+release_status_filter = lambda status: True
+if len(sys.argv) == 4:
+    release_status_filter_regex = re.compile(sys.argv[3])
+    release_status_filter = lambda status: bool(release_status_filter_regex.search(status))
+
+print(f"Will output the following types of items to {out_name}")
+print("undefined :", release_status_filter(""))
```

Comment:
Sounds like searching for `undefined` will always return True - `search` and `match` both succeed if any part of the string matches. If this is not desired, use `fullmatch()`.

Reply:
Removed undefined